pkg/odo/cli/project/create.go

```diff
@@ -81,7 +82,7 @@ func (pco *ProjectCreateOptions) Run() (err error) {
 
 	// If -o json has been passed, let's output the approriate json output.
 	if log.IsJSON() {
-		project.MachineReadableSuccessOutput(pco.projectName, successMessage)
+		machineoutput.ProjectSuccess(pco.projectName, successMessage)
```

Comment:
If the project creation was scessfull we should return `Kind` Project as defined in https://github.com/openshift/odo/blob/master/pkg/project/types.go#L7 representing the project that was just created

Reply:
The best way to do this would be to modify `project.Create` to return the newly created project. The underlying call in `client-go` is doing exactly that, we just need to convert it to our `Project` and return it